CHANGELOG.md

```diff
@@ -0,0 +1,202 @@
+### Version 0.0.10
+
```

Comment:
This patch includes all the hacks from the previous one, along with the following:

???